agent/fw_drupal_common.h

```diff
@@ -147,23 +147,23 @@ nr_status_t module_invoke_all_parse_module_and_hook_from_strings(
  */
 void nr_drupal_headers_add(zval* arg, bool is_drupal_7 TSRMLS_DC);
 
-
 #if ZEND_MODULE_API_NO >= ZEND_8_0_X_API_NO \
     && !defined OVERWRITE_ZEND_EXECUTE_DATA
 /*
- * Purpose: Before an invoke_all style call, adds the hook to that hook states stacks
+ * Purpose: Before an invoke_all style call, adds the hook to that hook states
+ * stacks
  *
- * Params  : 1. A zval holding a copy of the hook invoked, to be managed by the hook
- *              states stacks and freed by nr_drupal_invoke_all_hook_stacks_pop() after the
- *              invoke_all call completes
+ * Params  : 1. A zval holding a copy of the hook invoked, to be managed by the
+ * hook states stacks and freed by nr_drupal_invoke_all_hook_stacks_pop() after
+ * the invoke_all call completes
```

Comment:
This doesn't seem right. The same file shows `Purpose and `Params` docs strings, that span multiple lines, indented.

Reply:
I've manually fixed this here: fca6ace05bfc8294c8f017d6d77e743c6ed95ee4
For future reference, the reason for this is not due to a mis-configuration of clang-format. This is due to the line length limit (80 chars) and clang-format not being clever enough to match the indentation style in a docstring like this. These kinds of cases will require manual intervention.